sk-ctrl/src/tests/controller_test.rs

```diff
@@ -245,15 +253,19 @@ async fn test_setup_simulation_create_prom(test_sim: Simulation, test_sim_root:
 }
 
 #[rstest(tokio::test)]
-#[case::ready(true, false)]
-#[case::not_ready(false, false)]
-#[case::disabled(true, true)]
+#[case::ready(true, false, WebhookState::ReadyWithCaBundle)]
+#[case::not_ready(false, false, WebhookState::ReadyWithCaBundle)]
+#[case::disabled(true, true, WebhookState::ReadyWithCaBundle)]
+#[case::webhook_not_created(true, false, WebhookState::NotCreated)]
+#[case::webhook_ca_bundle_none(true, false, WebhookState::CaBundleNone)]
+#[case::webhook_ca_bundle_empty(true, false, WebhookState::CaBundleEmpty)]
 async fn test_setup_simulation_wait_prom(
     mut test_sim: Simulation,
     test_sim_root: SimulationRoot,
     opts: Options,
     #[case] ready: bool,
```

Comment:
We should rename `ready` to something more descriptive -- I initially thought this was referring to whether the webhook state was ready or not, instead of whether the rest of the setup had succeeded/whether it was ready to create the webhook.

Actually `ready_to_create_webhook` would be perfect lol.